packages/api/AlvTime.Persistence/Repositories/UserStorage.cs

```diff
@@ -67,5 +71,24 @@ public void UpdateUser(CreateUserDto userToBeUpdated)
 
             _context.SaveChanges();
         }
+
+        public async Task<Business.Models.User> GetUserFromToken(Token token)
```

Comment:
Does a token without an user makes sense? If not, maybe the business representation of a token should have a "user" property, and that this is populated in the TokenStorage when you retrieve the token?